brian2/equations/equations.py

```diff
@@ -930,6 +931,55 @@ def _sort_subexpressions(self):
             elif eq.type == PARAMETER:
                 eq.update_order = len(sorted_eqs) + 1
 
+    @property
+    def dependencies(self):
+        '''
+        Calculate the dependencies of all differential equations and
+        subexpressions.
+        '''
+        deps = {}
+        for eq in self._equations.values():
```

Comment:
Please add a few comments to understand why you have used this for loop and the purpose of putting a " ' " for the differential equations